bleach/sanitizer.py

```diff
@@ -488,9 +488,9 @@ def sanitize_uri_value(self, value, allowed_protocols):
             if ":" in new_value and new_value.split(":")[0] in allowed_protocols:
                 return value
 
-            # If there's no protocol/scheme specified, then assume it's "http"
-            # and see if that's allowed
-            if "http" in allowed_protocols:
+            # If there's no protocol/scheme specified, then assume it's "http" or
+            # "https" and see if that's allowed
+            if "http" in allowed_protocols or "https" in allowed_protocols:
```

Comment:
This is the minimal change to fix the bug. I do think the things we talked about are interesting as well, but I think I want to push them off so as to get 5.0.1 out the door.